### PR TITLE
Configure Cloudflare DNS via Settings menu

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/OS-1369) |

#### Main changes
Add a new page to the settings menu that adds/removes the Cloudflare DNS from the system.

Note: snapshot isn't 100% accurate - when using font size 9 on the device, the 'cloudflare dns' text fits in the screen.

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: pt_miniscreen/actions.py [hunk 1]
```diff
@@ -4,10 +4,13 @@
 from subprocess import Popen, check_output
 
 from pitop.common.command_runner import run_command
+from pitop.common.configuration_file import add_section, has_section, remove_section
 from pitop.common.sys_info import get_ap_mode_status, get_systemd_enabled_state
 
 logger = logging.getLogger(__name__)
 
+CLOUDFLARE_DNS_FILE = "/etc/resolv.conf.head"
```
[the remark this answers is not in the document]
Comment (jcapona) [reply pt_miniscreen/actions.py:12]: good one - thanks @m-roberts

**Review by angusjfw — APPROVED** [review]
